src/MuTalk-Tests/RandomMutantSelectionStrategyTest.class.st

```diff
@@ -0,0 +1,65 @@
+Class {
+	#name : 'RandomMutantSelectionStrategyTest',
+	#superclass : 'TestCase',
+	#category : 'MuTalk-Tests',
+	#package : 'MuTalk-Tests'
+}
+
+{ #category : 'comparing' }
+RandomMutantSelectionStrategyTest >> atLeastTwoDifferentMutantCollectionsIn: aCollection [
+	"aCollection is a collection of mutant collections. This method returns true if among these mutant collections, at least 2 are differents."
+
+	1 to: aCollection size - 1 do: [ :i |
+		i + 1 to: aCollection size do: [ :j |
+	"By iterating like this, we ensure to compare each mutant collection with every other collections
+	For example, when i = 1, 2 <= j <= 5, when i = 2, 3 <= j <= 5 (collection 2 and collection 1 have already been compared previously), etc"
+			(self
+				 sameMutationOrderIn: (aCollection at: i)
+				 and: (aCollection at: j)) ifFalse: [ ^ true ] ] ].
+	^ false
```

Comment:
Why not putting just all collections into a set and counting the number of elements?

Reply:
Ohh ok, I understand.
I tried to use a set before, but I used `addAll:` to add the collections in the set. But by doing so, it added all the mutants and not the collections themselves. Then, adding any collection past the first didn't change the content of the set, since all collections contain the same mutants, but in different orders.
But yes, adding the collections themselves instead could work.